### DISCUSSION TITLE
How do I stop seeing feedback from some users?

### DISCUSSION THREAD

**Discussion by leoheck** [community/community#27761] [Other Feature Feedback, Questions, & Ideas]
Question in the title.

**Comment by jfullstackdev** [comment]
what feedback is that ? 

block the user

also there is no need to repeat the post so many times

**Reply by leoheck** [reply]
I dont want to block the user. I just don't use his feedback. I don't care what they are doing. I don't want to just stop any way this person has to see my stuff or interact with me. I just don't want to see them breathing, pooping, eating, e.ve.r.y. fucking time.

**Comment by ihnatenkoo** [comment]
Hello, Leonardo. You can set up **Filter your feed**

![ффф](https://user-images.githubusercontent.com/89586345/182011877-1c5385f9-51f4-4820-bd33-6ba8b7e77370.png)

**Comment by leoheck** [comment]
Hello Dimitrisky

I don't want to block the user. I just don't use his feedback. And this is also polluting the whole feedback. I don't care what they are doing. I don't want to just stop any way this person has to see my stuff or interact with me. I just don't want to see them breathing, pooping, eating, e.ve.r.y. fucking time.

**Comment by jfullstackdev** [comment]
this is the same user at the other post, seems there is no solution to his problem

you are too annoyed by those users yet you posted here so many times too,

**Reply by leoheck** [reply]
Yeh, because this shit was introduced by Microsoft. It was good before. Now Microsoft is creating these useless "innovations". And I, as a single user have no voice. So I am doing my best to be at least listened to. If they create a shitty feature, they have to create a solution to help users to decide what is better for them. These "social media"  is full of immature useless new features. This is why people hate these things. Github is not a "social media". It is for code, and to share code. Not to elevate people. For that, we have other places.

**Reply by leoheck** [reply]
> you are too annoyed by those users yet you posted here so many times too,

And yeh, see how annoying this is? You completely got my point here.
